bin/crystal

```diff
@@ -141,19 +141,21 @@ CRYSTAL_DIR="$CRYSTAL_ROOT/.build"
 export CRYSTAL_PATH=lib:$CRYSTAL_ROOT/src
 export CRYSTAL_HAS_WRAPPER=true
 
+CRYSTAL="${CRYSTAL:-"crystal"}"
```

Comment:
Note to self: I think I’m missing to export this variable. So if the program calls again bin/crystal with an overrode value by makefile argument (not env var) it would be picked again.